jxl/META.yml

```diff
@@ -0,0 +1,3 @@
+spec: https://jpeg.org/jpegxl/workplan.html
+suggested_reviewers:
+  - TBD
```

Comment:
Would you be willing to review changes to this directory going forward? If so you can put yourself here, and I can make you a WPT reviewer.

Reply:
Sure, I'm happy to review changes going forward!
Added myself here.
Should there also be a second name, in case changes are coming from me?

Reply:
If you have a second name, sure, but if you don't have one, you can always ask someone else. This field is only for adding reviewers automatically, anyone in the reviewers team can still review.